src/node/node_class.hpp

```diff
@@ -266,15 +267,25 @@ static inline std::vector<napi_value> napi_get_arguments(const Napi::CallbackInf
 	return arguments;
 }
 
-// The static class variable must be defined as well.
+//This forces initialization of the static member in sequence
```

Comment:
Pedantically, the technique you are using does the opposite of what the comment implies. It *avoids* forcing the members to be initialized (at the normal point), and instead defers initialization until the first time any of these functions are called.

I'd just delete the comment since I don't think it adds much. Really, you should probably comment when you *don't* use this technique to explain why it is OK, rather than commenting when you do 🙂

Reply:
thanks. It guess it forces the compiler rather, to defer the initialization. But maybe this comment is confusing. will remove it